src/screens/AnswerScreen/AnswerScreen.js

```diff
@@ -0,0 +1,293 @@
+                                    placeholder = "Ask your query, by addressing your problem clearly!"
+                                    value = {this.state.answerQuery} 
+                                    onChangeText={(answerQuery) => this.setState({answerQuery})}
+                                />
+                                {setDisplayImage ? 
+                                <TouchableOpacity onPress={this.showAlert} >
+                                    <Image source={{uri: setDisplayImage}} style={styles.ImageStyle}/>
+                                </TouchableOpacity>
+                                : null}
+                                <TouchableOpacity 
+                                style={styles.AnswerButtonStyle}
+                                onPress={this.addAnswer}
+                                >
+                                    <Text style = {styles.AnswerStyle}>Answer</Text>
+                                </TouchableOpacity>
+                            </View>
+                        </View>
+                        <View style={{flexDirection: 'row', alignSelf:'flex-end',}}>
+                        {setDisplayImage ? null : setImage ? 
+                        <Progress.Bar progress={transferred} width={230} style={styles.ProgressBarStyle}/> : null}
+                        {setDisplayImage ? null : setImage ? 
+                        <Icon style={styles.IconStyle1} size={30} name={'cloud-upload-outline'} onPress={this.uploadImage}/> 
+                        : 
+                        <Icon style={styles.IconStyle1} size={30} name={'image-outline'} onPress={this.takePhotoFromLib}/>}
+                        </View>
+                    </View>
+                    <View style={styles.DividerView1} />
+                        <FlatList
+                            data={answerDetail}
+                            renderItem = {({item}) => {
+                                return(
+                                    <View style={{marginVertical: 10}}>                                        
+                                        <View style = {styles.QuestionStyle}>
+                                            <View style={styles.nameHeadlineStyle1}>
+                                                <Text style={styles.userAnsStyle}>{item.userAnswer}</Text>
+                                                <Text style={styles.ansTimeStyle}>{item.answerTime}</Text>
+                                            </View>
+                                            <View>
+                                                <Text style={styles.Q_ansStyle}>{item.answerQuery}</Text>
+                                                { item.answerImage ? 
+                                                <TouchableOpacity onPress={() => Linking.openURL(item.answerImage)}><Image source={{uri: item.answerImage}} style={styles.ImageStyle1} /></TouchableOpacity> : null}
```

Comment:
set character limit for a single line(including spaces/tabs also)

Reply:
Updated👍🏻